src/tools/chdman.cpp

```diff
@@ -2818,7 +2910,8 @@ static void do_extract_cd(parameters_map &params)
 		// delete the output files
 		output_bin_file.reset();
 		output_toc_file.reset();
-		osd_file::remove(*output_bin_file_str);
+		for (auto output_bin_filename : output_bin_filenames)
+			osd_file::remove(output_bin_filename);
```

Comment:
Can you use `for (auto const &output_bin_filename : output_bin_filenames)` here to avoid unnecessary copies?